BlueprintUI/Sources/Internal/Equivalency.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+
+// A context in which to evaluate whether or not two values are equivalent.
+public enum EquivalencyContext: Hashable, Sendable, CaseIterable {
```

Comment:
I am wondering if we'd need this at all if we returned to tracking / comparing only the environment values that are read by the element in question, vs. comparing everything